.travis.yml

```diff
@@ -3,19 +3,24 @@ env:
+  - secure: 14MqW8PG56KvOnMGTV8jKrzMKjsR5CF6vUW+xLBYls61e1ptX86ElLd0SzaxEkoyLUQ7RNdWDNyW0tKTlpUdgJU1osdYTPMl9L/6nc2i0vH78MJVwN0DJBmou2f3mp21foOoKQVkYc8nYfNsNvXb0DAddRH2M7xcFNS++88qoWRjw6ACMnrH1vFWm1icI17Wd8zhjZqFsOeJIF4rnzcavZMZnhxd/gp+QTfyoh1QtPnlO6h3wYprhKjGubPW+j34N7y0bskuPLHDpjKTeXWZmfvObO9S2C7UZ0CjtOxj1jTyXloar50m/qegkO8EVvOnxFmDhMXR647tManCme3/AxYtwsOpBY/fbs5kp4TMoZXlXQBJLGPTXDlTlRYAtRU8no7pazMp4xP4TvGk5HQ4UIztIZS/ZeHYTIGF/NQALU+XbQcQtT3wNn5LZxpod+JSaad1VJDaM7YUuh2wQroABalX00tGgvkXm3bL1yxIs82OonSA7jGCwaFmAU8rO/ZTCLC9GbO1+BL09J1LBxp+wIfqYiahSnTelxUBeXLkn4ktVn/fk2GZBUaBcTv/YYRoNOc6BvJkB3acsK4GDoDrhQEbkv9H6WDJ6e43vTsbfKV72smJ5JvXJZvB23VUFqFVVGN5tLMDE1e+mkwHFZ4aZ0q3hcU9nl7v2yXoVTObUPQ=
+  - secure: JEYD3nhS8hgZI8Mcz4cO/w5J+EMwK0qU84cqHimP27YguySRgWtb6OyVQ1Jvt4bwPDWtf8RqqCYRRRUfxFtv3KtDUZyYRspcvQoCDmakea3HF2dzAe+8xFYa5KmHzMI1Z1pFND06/chdIOUMEZ0OMxc10SJzDNdOw2ZWolWXqBpb9PBV/mQz9XQsGjuzDlSfpsnLetP77seXVF7bqXCK0dqUukLqO1llyF06s8FsDqLDYyA3cZ8TlhRjCZm8qbtvI+6yC490f42VWWXIDKlD+a5hVEhOo/N5V50x1r0ltOipAIjZLxRxNI9FVk7c4FdQ5qw+uUuciW9tgM47kUCx7zWtYKJDlThnQf/RKSGhkbOoYeHzggqjkBdPe2jHIRPhzaL6VuLlM8brZa58hd91cETvRoshwIlJpG9UaRn+hr9HdPR3HQsTFHsxtE5fhRFPv4DlVl9b5HByZDydmJHuGyoYmR4LmTJqpvxnMZyisoUZdMhBXdelleaPQKhM2oDKBYD2qIHRy/Fx+tg5AKG7ieWVRrI3CTQ/IbwaHe/p4U4AO9EXnn8O3I/2EJZnzKwNjNkgrMx4zkEpXyKs2OdR0LRaF26Ifb42f4m3jfDe6hzNNZcrQpfwKJQktXf+t7bfMp/97WqV8aaNcW8GzWJBRZIy0xClUHLUYUDqaKJF5G0=
+  - secure: I/updrnUdSztTqVUOo+8Ap59fk5Ejk6R+YQPK+Pm8Ox8heMTrCa1aFbHnjwUr9GRLKmOBsu2hJ3fqg6ysj2rzVo1f75GPTun2B5nLvM8SVMaXfZA+dem44KXsYsUmeYQLM6iUpcS2fImwCvErr5Mtc4wQyABlkM/YHaKcb/jxDSjWYo+vbOLEhmt/VbQMCFLBcQQefw5dznTNqGvovrvJYaTik5VECNeh9kxNhPdbdVAYMyBWiCkTGORRC+ZPqYKJL2ZTFSYQAEVctoKM5bPPNHULCu+vk+jT8eCVGb8tBmfrH0yzUI+ER3EqrN62vnsqr2JajhjUHkyf2baEVRDRmS1oBAcxbztP2MDXezPPdAjbWyFgOj6Sf8udQE+j4f4YP7JNythcbydXQ9zJ4qNfentuGIWNv4hy5xXn4CGPQlbhRrlgKWO1mCmiHKesEZ+4jVkJ21TUKTvE6kssogjltMdJOQxUVWQnCihoalPhXU0YUK9kQ3vFKcteO93Lk8lK3ZSZPeesyV7gXtWSoKyGJTYOjKUAXpQ17g1II93BKFPkxt5YeLbLYQ33YEpw3QAih3xCaLoSdyiiHpCopGGb/mJ3KoPoyOK/TLlcT+KIW5FP2otU1z5cXAxoYWXkM7g4BhVhc7g7zPQeE/dzVaR/Ryoug2M4aONuwRQKtw3PS4=
 sudo: required
 language: java
 services:
 - docker
 script:
-- mvn --quiet --batch-mode install
+- mvn --quiet --batch-mode verify
 - docker build -t salesforce/dockerfile-image-update .
 - echo git_api_token=${ITEST_GH_TOKEN} > `pwd`/itest.env
 - export user_itest_secrets_file_secret=`pwd`/itest.env
 - docker-compose up
 after_success:
 - mvn coveralls:report
-- if [ "${TRAVIS_BRANCH}" == "master" ]; then 
-      docker login -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}";
-      docker push salesforce/dockerfile-image-update; 
-  fi
+deploy:
+  provider: script
+  script: "./travis.deploy.sh"
```

Comment:
`.travis`, not ./travis

Reply:
Fixed in d8fb072657a729c83c472065c3861347f5528fd9